README.md

```diff
@@ -1,6 +1,6 @@
 # Samsu Assasins
 
-By: George Teo [other contributors put your name here]
+By: George Teo [Scarlett Li]
```

Comment:
This is a good opportunity to understand the flow of comments and changes. 

Please remove the [] around your name and submit the pull request again.